standard-dev.cfg

```diff
@@ -0,0 +1,45 @@
+# Standard development config to be used in standardized policies.
```

Comment:
Oh, you're right. I have moved the documentation to the readme and I've enriched it ( 4b1bc34 ).